fem/tmop.cpp

```diff
@@ -4324,7 +4607,11 @@ void TMOP_Integrator::ComputeNormalizationEnergies(const GridFunction &x,
          {
             if ((*surf_fit_marker)[dofs[s]] == true)
             {
-               surf_fit_gf_energy += sigma_e(s) * sigma_e(s);
+               const IntegrationPoint &ip_s = ir_s.IntPoint(s);
+               Tpr->SetIntPoint(&ip_s);
+               surf_fit_gf_energy += surf_fit_coeff->Eval(*Tpr, ip_s) *
```

Comment:
I thought user-provided `Coefficients` were not part of the normalization. They would get cancelled out and have no effect.